.coveragerc

```diff
@@ -2,7 +2,6 @@
 omit = 
     *wsgi.py
     */admin/*
-    */teachers_digital_platform/migrations/*
```

Comment:
I think we can delete this config file.

Reply:
I think we're going to keep stubs for testing Python so that any Python added to the cartoon can be tested.